ionoscloud/resource_dbaas_mongodb_cluster.go

```diff
@@ -28,6 +29,7 @@ func resourceDbaasMongoDBCluster() *schema.Resource {
 		Importer: &schema.ResourceImporter{
 			StateContext: resourceDbaasMongoClusterImport,
 		},
+		CustomizeDiff: errorOnMongoVersionDowngrade,
```

Comment:
@rmocanu-ionos please check if the following tests exist and if not, please write them:
- test that version upgrade is successful
- test that the version downgrade shows an error